plugins/catalog-backend-module-azure/src/processors/AzureDevOpsDiscoveryProcessor.ts

```diff
@@ -138,11 +145,18 @@ export function parseUrl(urlString: string): {
   project: string;
   repo: string;
   catalogPath: string;
+  branch: string;
 } {
   const url = new URL(urlString);
   const path = url.pathname.slice(1).split('/');
 
   const catalogPath = url.searchParams.get('path') || '/catalog-info.yaml';
+  let branch = url.searchParams.get('version') || '';
+
+  if (branch.startsWith('GB')) {
+    // DevOps prefixes branch names with 'GB' in URLs
```

Comment:
As far as I can tell, this is always true. In Azure DevOps, you can reference different things with the `version` param, such as tags which would then prefix it with `GT`, or specific commits which uses `GC` (I assume "Git Branch", "Git Tag" and "Git Commit"). I haven't tested the Code Search feature with referencing specific commits or tags though, and my assumption is that it wouldn't work since you can only add branches, not tags or commits, to the search index in Azure DevOps.

The reason we have to remove the prefix here is that the Code Search feature just accepts the branch name and won't return any hits if the prefix is included. Since the URL in the address bar in Azure DevOps always includes the prefix, not doing this would mean that directly copy-pasting the location URL would not work, which could be confusing.

So not prefixing `GB` to the branch name in the `azure-discovery` location configuration should actually work as well, but I decided not to explicitly document that to keep it simple and conform to the Azure DevOps way of doing things as much as possible, to avoid adding unnecessary abstractions that might be constraining later.